### ISSUE TITLE
Show preview images within asset list 

### ISSUE THREAD

**Issue by gerhard-boden** [neos/neos-ui#1305]
In the old UI we had preview images within asset list items. Would be nice to port that to the new UI as well.

relates: #1296

**Comment by dimaip** [comment]
Would be nice to tell MultiSelect to use the option renderer for showing the selected options. That would solve this issue as well.

**Comment by dimaip** [comment]
Blocked by #1362

**Comment by dimaip** [comment]
Can be closed now, we have a separate issue for making them bigger.
